Added logging documentation (#285)
strimzi/strimzi-kafka-bridge: documentation/book/logging.adoc

= Logging

> This section provides information on loggers and how to configure log levels.

Reviewer: Suggest removing this line.

> The bridge allows to set a different log level for each of the operations defined by the related OpenAPI specification.

Reviewer: ```suggestion
The Kafka bridge allows you to set a different log level for each operation that is defined by the related OpenAPI specification. 
```

Reviewer: Insert a blank line (new paragraph).

> Each operation matches a corresponding API endpoint on which the bridge gets requests from the HTTP clients.

Reviewer: ```suggestion
Each operation has a corresponding API endpoint through which the bridge receives requests from HTTP clients.
```

> Changing the log level on each of them allows to have a more or less fine grained logging information about the incoming HTTP requests and the outgoing HTTP responses.

Reviewer: ```suggestion
You can change the log level on each endpoint to produce more or less fine-grained logging information about the incoming and outgoing HTTP requests.
```

> The loggers are defined inside the `log4j.properties` file which by default has the following configuration for `healthy` and `ready` endpoints.

Reviewer: ```suggestion
Loggers are defined in the `log4j.properties` file, which has the following default configuration for `healthy` and `ready` endpoints:
```

```
log4j.logger.http.openapi.operation.healthy=WARN, out
log4j.additivity.http.openapi.operation.healthy=false
log4j.logger.http.openapi.operation.ready=WARN, out
log4j.additivity.http.openapi.operation.ready=false
```

> All the other operations are by default at logging `INFO` level.

Reviewer: ```suggestion
The log level of all other operations is set to `INFO` by default.
```

> The format for each logger is the following.

Reviewer: ```suggestion
Loggers are formatted as follows:
```

```
log4j.logger.http.openapi.operation.<operation-id>
```

Where `<operation-id>` is the identifier of the specific operation.

> Following the list of the operations defined by the OpenAPI specification.

Reviewer: ```suggestion
Following is the list of operations defined by the OpenAPI specification:
```
